example.py

```diff
@@ -37,10 +37,16 @@ async def connect(args: argparse.Namespace, session: ClientSession) -> None:
     """Connect to the server."""
     websocket_url = args.url.replace("http", "ws") + "/api/websocket"
     async with HomeAssistantClient(websocket_url, args.token, session) as client:
-        await client.subscribe_events(log_events)
         # start listening will wait forever until the connection is closed/lost
-        await client.start_listening()
+        asyncio.create_task(log_exceptions(client.start_listening()))
+        await client.subscribe_events(log_events)
+        await asyncio.sleep(60)
 
+async def log_exceptions(awaitable):
+    try:
+        await awaitable
+    except Exception as e:
+        LOGGER.error(e)
```

Comment:
Exceptions need to be handled, not ignored.
Also if you want to log them, maybe a stack trace is convenient.
So a simple LOGGER.exception would be better, especially for unhandled exceptions

Reply:
The `await listener_task` is a much better solution, using that.